.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
```

Comment:
I actually really expect there to be master & release (patch) branches in this repo.